src/cli/init/templates.ts

```diff
@@ -17,7 +17,31 @@ import { InitOptions } from "../types";
 
 export const { dependencies, devDependencies, peerDependencies, scripts, version } = packageJSON;
 
-export function genPkgJSON(opts: InitOptions, pgkVerOverride?: string) {
+type peprPackageJSON = {
+  data: {
+    name: string;
+    version: string;
+    description: string;
+    keywords: string[];
+    engines: { node: string };
+    pepr: {
```

Comment:
Good work, just to let you know here be dragons on this story, the implementation is good though. I am glad we are getting it strongly typed.

We are missing a few config options that can be in the pepr section of `package.json`.

check out [customization](https://docs.pepr.dev/main/user-guide/customization/). Looks like we need `rbac` (type `PolicyRule[]`) and `rbacMode`.

Reply:
Agreed, added